File: app/src/main/java/mx/dev/franco/automusictagfixer/UI/AudioHolder.java
package mx.dev.franco.automusictagfixer.UI;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;

public abstract class AudioHolder extends RecyclerView.ViewHolder {
    public ImageView cover;
    public AudioHolder(View itemView) {
        super(itemView);
    }
}

File: app/src/main/java/mx/dev/franco/automusictagfixer/ui/AudioHolder.kt
package mx.dev.franco.automusictagfixer.ui

import android.view.View
import android.widget.ImageView
import androidx.recyclerview.widget.RecyclerView

abstract class AudioHolder(itemView: View?) : RecyclerView.ViewHolder(
    itemView!!
) {
    lateinit var cover: ImageView
}